### PR TITLE
Add Tribecastudios.co.uk.xml

### PR DESCRIPTION
**Pull request by wjt**
Just a local gym. I didn't notice any mixed-content warnings etc. when signing into my account, booking a session, and generally poking around the site.

### CODE REVIEW

**Comment by J0WI** [comment]
Please also add `blog.tribecastudios.co.uk`

**Comment by wjt** [comment]
Chrome, Firefox, Epiphany and HTTPie all report an invalid response (not even a certificate error) when accessing https://blog.tribecastudios.co.uk/ , so I don't think that's a good idea.

**Comment by J0WI** [comment]
We usually try list each host, either as a target or otherwise as `Nonfunctional hosts in *.tribecastudios.co.uk:`. This will make maintenance easier for us and sysadmins can see the issue in the HTTPS Everywhere atlas.

**Comment by wjt** [comment]
Ah, I understand now, thanks. I'll take a look.

**Comment by wjt** [comment]
How's that? I've also contacted the website to suggest they force HTTPS anyway.

File: src/chrome/content/rules/Tribecastudios.co.uk.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Tribecastudios.co.uk">
+	<target host="tribecastudios.co.uk" />
+	<target host="www.tribecastudios.co.uk" />
+
+	<!-- accessing the bare domain with https: redirects to http://www.zingfit.com/
+	     so explicitly add www.
```
Comment (J0WI) [inline src/chrome/content/rules/Tribecastudios.co.uk.xml]: Please see https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects

**Comment by J0WI** [comment]
That's great, thanks!

**Pull request merged**